subsys/logging/Kconfig.backends

```diff
@@ -56,6 +56,12 @@ config LOG_BACKEND_UART_OUTPUT_DICTIONARY_BIN
 
 endchoice
 
+config LOG_BACKEND_UART_OUTPUT_DEFAULT
```

Comment:
I would recommend adding a comment here indicating that the numbering for LOG_BACKEND_UART_OUTPUT_TEXT,  .... be consistent across all backends in the Kconfig AND that their values must match the LOG_OUTPUT_xxx type in the C files.

Reply:
Added the comment indicating the details.